service.libraryautoupdate/resources/lib/utils.py

```diff
@@ -0,0 +1,35 @@
+import xbmc
+import xbmcgui
+import xbmcaddon
+import xbmcvfs
+
+__addon_id__= 'service.libraryautoupdate'
+__Addon = xbmcaddon.Addon(__addon_id__)
+
+def check_data_dir():
+    if(not xbmcvfs.exists(xbmc.translatePath(data_dir()))):
+        xbmcvfs.mkdir(xbmc.translatePath(data_dir()))
+
+def data_dir():
+    return __Addon.getAddonInfo('profile')
+
+def addon_dir():
+    return __Addon.getAddonInfo('path')
+
+def log(message,loglevel=xbmc.LOGNOTICE):
```

Comment:
Please change this default to debuging and only log the messages really required with lognotice.
```suggestion
def log(message,loglevel=xbmc.LOGDEBUG):
```